### DISCUSSION TITLE
Physx world gravity direction change

### DISCUSSION THREAD

**Discussion by byCyrus** [o3de/o3de#11457] [Q&A]
Any chance change world gravity direction in runtime?
![Screenshot_1](https://user-images.githubusercontent.com/3614390/186012848-d7503290-6f29-4928-b1a4-e8f73740dec9.png)

**Comment by greerdv** [comment]
Yeah, you can change gravity (including changing the direction) with `SetGravity` on the physics scene interface.

**Reply by greerdv** [reply]
It looks like that's not exposed to scripting though. I've created https://github.com/o3de/o3de/issues/14186 to get it added.
